primitives/src/ticker.rs

```diff
@@ -51,31 +51,13 @@ impl AsRef<[u8]> for Ticker {
 }
 
 impl TryFrom<&[u8]> for Ticker {
-    type Error = Error;
+    type Error = ();
```

Comment:
If it can't fail, it should implement the `From` trait not `TryFrom`, but why were the checks removed?

Reply:
We have a lot of test/benchmark code that uses `try_from(name).unwrap()`.

Maybe it would be better/cleaner to replace this fn with a safer `from_name()` function.